### PR TITLE
Semi-port of banners from Polaris.

### PR DESCRIPTION
**Pull request by MistakeNot4892**
## Description of changes
Adds banner frames and banners. Banner frames are made from wood/steel/etc, banners are made at the textiles fabricator. You can hang banners on banner frames.

image

The above are presets, banners and frames are set up to take colour from material.

Someone might want to go over the icons at some point, I greyscaled them very roughly.

## Why and what will this PR improve
Decorative item used by a few downstreams, made a bit more modular and interactive for Neb.

## Authorship
Myself for the code.
Icons originated on Polaris before I greyscaled them (poorly), I'm not sure where the are from originally. I remember seeing them on Paradise way back in the day.

## Changelog
:cl:
add: You can now build banner stands and print banners at a textiles fabricator, or craft one from cloth.
/:cl:

### CODE REVIEW

**Comment by MistakeNot4892** [comment]
Going to unfreeze this as it's small and the potential for bugs is minimal.

**Review by comma — APPROVED** [review]

File: mods/content/government/items/clutter.dm [hunk 1]
```diff
@@ -7,4 +7,40 @@
...
+
+/obj/structure/banner_frame/virgov
+	banner = /obj/item/banner/virgov
+
+/datum/fabricator_recipe/textiles/banner/virgov
+	path = /obj/item/banner/virgov
+
+/obj/item/banner/virgov
+	name = "\improper VirGov banner"
+	hung_desc = "The banner is emblazoned with a white VirGov seal."
```
Comment (out-of-phaze) [inline mods/content/government/items/clutter.dm:38]: What is VirGov? Was this an unintentional port when you cherry-picked it from your Polaris branch?

**Review by out-of-phaze — CHANGES_REQUESTED** [review]
Seems there may have been an accidental port? I don't see any other references to VirGov in the government modpack. The only current references to Vir in the code are short for Virology.

File: mods/content/government/items/clutter.dm (hunk 1 above)
Comment (MistakeNot4892) [reply mods/content/government/items/clutter.dm:38]: Not unintentional, just lazy. I can trim it out if it matters.

**Review by out-of-phaze — APPROVED** [review]

**Pull request merged**